tests/sources/test_cppcheck.py

```diff
@@ -163,9 +172,11 @@ def test_projects_with_cppcheck(self):
             'projects/vehicles/controllers/ros_automobile/include'
         ]
         skippedfiles = [
-            'projects/robots/robotis/darwin-op/plugins/remote_controls/robotis-op2_tcpip/stb_image.h'
+            'projects/robots/robotis/darwin-op/plugins/remote_controls/robotis-op2_tcpip/stb_image.h',
+            'projects/robots/epfl/lis/plugins/physics/blimp_physics/utils.h'
```

Comment:
Just for info, why this file now needs to be ignored?
Was it working correctly for macOS and Windows cppcheck before, isn't it?

Reply:
This is because we now only run cppcheck on the source code with `_WIN32`, `__linux__` and `__APPLE__` preprocessor variables, while before it was running all possible configurations, including other variables. In `utils.h`, there is an error thrown if neither `dSINGLE` or `dDOUBLE` are defined. This is why this file is skipped now.